src/src/tx_main.cpp

```diff
@@ -235,6 +235,49 @@ void ICACHE_RAM_ATTR Generate4ChannelData_11bit()
   Radio.TXdataBuffer[6] += CRSF_to_BIT(crsf.ChannelDataIn[6]) << 1;
   Radio.TXdataBuffer[6] += CRSF_to_BIT(crsf.ChannelDataIn[7]) << 0;
 #endif
+
+#ifdef USE_DYNAMIC_POWER
```

Comment:
Replace with #ifdef DYNAMIC_POWER_CHANNEL if moved to user_defines?

Reply:
So can adding featire dynamic RF 25hz - 200hz?